### ISSUE TITLE
2016-09-08 09:11:21,648 [     alarm_manager] [  ERROR] Error while processing request: 'Telegram_Alarm' object has no attribute 'disable_map_notification'

### ISSUE THREAD

**Issue by roflinger** [kvangent/PokeAlarm#208]
## Problem Description

2016-09-08 09:11:21,648 [     alarm_manager] [  ERROR] Error while processing request: 'Telegram_Alarm' object has no attribute 'disable_map_notification'

## Your Environment

Operating System:  win 10 64bit
Python Version: (use `python -V`)  2.7
PokemonGo-Map startup command: `<cmd>`  
PokeAlarm startup command: `<cmd>` >python runwebhook.py

Did the issue persist after restarting the script?: yes

## Console Output

```
	<2016-09-08 09:11:21,648 [     alarm_manager] [  ERROR] Error while processing request: 'Telegram_Alarm' object has no attribute 'disable_map_notification'>
```

## Your Alarms Config

```json
    <default with only telegram enabled and entered bot/chatid, also set pidgey true>
```

## Troubleshooting Checklist
[x ] Most recent version of PokemonGo-Map
[x ] Most recent version of PokeAlarm
[x ] Checked the Wiki page for service in question

**Comment by kvangent** [comment]
@roflinger should be fixed with a4c45ba6422ed0db67898754ffcdefcf73deb4f4. If you still have issues let me know.

**Comment by roflinger** [comment]
yep, works - thx!
